docs/stencil-docs/localization/localization-tutorial.md

```diff
@@ -0,0 +1,127 @@
+# Localization Tutorial
+<div class="otp" id="no-index">
+
+### On this page
+- [Getting started](#getting-started)
+- [Creating translation keys](#creating-translation-keys)
+- [Updating browser settings](#updating-browser-settings)
+- [Resources](#resources)
+
+</div>
+
+You have the option of localizing your theme for desired target languages. This tutorial describes the case of localizing a storefront for multiple languages. The translated values are displayed depending on the language selected in your browser. By the end of this tutorial, you should be familiar enough to localize headers, footers, buttons, payment methods, or any part of your theme.
+
+### Prerequisites
+
+For this tutorial, you will need to use Stencil CLI and use [Browsersync](https://github.com/bigcommerce/browser-sync) to serve up a live preview of a theme in development.
+
+This article assumes you have familiarity with the following concepts:
+
+* [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil)
+* [Creating a Stencil CLI Token](https://support.bigcommerce.com/s/article/Store-API-Accounts)
+* [Downloading and Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload)
+* [Serving a live preview](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#serving-a-live-preview)
+
+
+## Getting started
+1. After downloading and extracting your Stencil theme, open the folder containing your theme files and navigate to the `/lang` subfolder. 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+> **NOTE:** The `/lang` subfolder includes the `en.json` file. The `en.json` file and your language file must be present for a localized Stencil theme to work. Each language requires its own JSON file. 
+
+</div>
+</div>
+</div>
+
+2. Use the `en.json` file and copy over the necessary key-value pairs. Update the values to create your language files.
+
+ ar.json
+```json                                                                                           
+{                                                                                       
+"header": {                                                                   
+   "welcome_back": "{name} ,مرحبا",
+   "skip_to_main": "تخطي إلى المحتوى الرئيسي"
+    },                                                                                 
+}                                                                                          
+```
+
+fr.json
+```json
+{
+"header": {
+  "welcome_back": "Bienvenue, {name}",
+  "skip_to_main": "aller au contenu principal"
+   },
+ }
+```
+3. Name your translation file based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language and region codes.
```

Comment:
This should be mentioned earlier since the file has already been created at this point.